scheduler-server/src/test/java/gov/nasa/jpl/aerie/scheduler/server/services/TypescriptCodeGenerationServiceTest.java

```diff
@@ -58,6 +62,9 @@ void testCodeGen() {
                  + "        peelDirection: (\"fromTip\" | \"fromStem\")\n"
                  + "      }) => PeelBanana\n"
                  + "  }\n"
+                 + "  var ActivityTypes: {\n"
```

Comment:
We can define the enum here too using the same syntax I believe

`enum ActivityType {
...
}`

Reply:
You're comment about plurality in the naming is coming to mind...

Reply:
Yeah, I'm wondering if there's a clean way to consolidate the `const` declaration with the `var` declaration for the same identifier... drawing blanks at the moment.

Reply:
Oh wait, would `var ActivityTypes: ActivityType` work? I'l go give that a try.

Reply:
Ah, the incantation `var ActivityTypes: typeof ActivityType` seems to work :+1:

Reply:
Now I'm wondering if we could do something similar with `ActivityTemplates`:

```typescript
const ActivityTemplatesAlias = ActivityTemplates;
declare global {
  var ActivityTemplates: typeof ActivityTemplatesAlias
}
```

> Note: the alias is needed because simply writing `var ActivityTemplates: typeof ActivityTemplates` gives the error: `'ActivityTemplates' is referenced directly or indirectly in its own type annotation`.

Pro: "single source of truth"
Con: I could conceive of us wanting to export the types without the implementations (e.g. to reduce file size). Perhaps having them separate now would make that easier to do in the future.

Reply:
I doubt we will reach the point of needing to reduce file size on this, and if we want to be able to split out the definitions from the implementations (which we very well may way to do) we'd need to split it into different files (or at least be able to generate a separate declarations file) so I don't think we should worry about it in this PR.